RESideMenu/RESideMenuController.m

```diff
@@ -711,6 +714,13 @@ - (void)resetContentViewScale
     self.contentViewContainer.frame = frame;
 }
 
+- (void)performInitialAppearanceTransitionCallsForControllerIfRequired:(UIViewController *)controller {
+    if (controller.childViewControllers.count > 0) {
+        [controller beginAppearanceTransition:YES animated:NO];
```

Comment:
Is view did load really the best place to call this? Because technically the view isn't appearing here, and then when you actually display the left menu you're going to get another appearance transition call without a disappear transition in between?